content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Operator Quick Start Evaluation Install
+description: Instructions to install Istio in a Kubernetes cluster for evaluation.
+weight: 9
+keywords: [kubernetes, operator]
+aliases:
+---
+
+This guide installs Istio using the standalone Istio Operator. The only dependencies
+required are a supported Kubernetes cluster and the `kubectl` command. This
+installation method lets you quickly evaluate Istio in a Kubernetes cluster on
+any platform.
+
+To install Istio for production use, we recommend using the [Helm Installation guide](/docs/setup/install/helm/)
+instead, which is a stable feature.
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/#downloading-the-release).
+
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/).
+
+1. Check the [Requirements for Pods and Services](/docs/setup/additional-setup/requirements/).
+
+## Installation steps
+
+1. Install the Istio Operator using the default profile:
+
+    {{< text bash >}}
+    $ kubectl apply -f https://preliminary.istio.io/operator.yaml
+    {{< /text >}}
+
+1. (Optionally) change profiles from the default profile to one of the following profiles:
```

Comment:
Can you install a non-default profile directly? If so, consider merging these two steps. User would just pick one of the profiles (tabs) and then install... instead of installing default and then switching.

Reply:
good idea - I will test it.  I am not immediately sure replicating the manifest is a good idea, but it may not cause any change to the objects except the CR.

Reply:
I think we want to take this approach in the future if it works.  Currently profile switching is not functional.  A few devleopers are looking at it and I think these instructions will help them hone in on the problems.  I don't want to complicate that with extra manifest information (just changing of CRs should change the deployment properly) - but will do so once we are confident in the profile switching functionality.